### PR TITLE
RF 03.01 | Multi-configuration project > POM

### PR DESCRIPTION
**Pull request by goluublue**
https://trello.com/c/aodGBFTc

### CODE REVIEW

**Review by evanmaiqa — COMMENTED** [review]
Trello Card?

**Comment by goluublue** [comment]
> Trello Card?

updated now, wanted to wait until checks were guaranteed to be passed

**Review by evanmaiqa — APPROVED** [review]

File: src/test/java/school/redrover/MultiConfigurationTest.java [hunk 1]
```diff
@@ -74,69 +73,58 @@ public void testRenameMultiConfigurationProjectFromDashboard() {
...
 
-        Assert.assertEquals(getDriver().getTitle(), "Dashboard [Jenkins]");
+    @Test(dependsOnMethods = "testEnabledMultiConfigurationProject")
+    public void testProjectPageDelete(){
+        MainPage deletedProjPage = new MainPage(getDriver())
+                .navigateToProjectPage()
+                .deleteProject();
 
-        List<WebElement> projects = getDriver().findElements(By.xpath("//a[@href='job/" + MULTI_CONFIGURATION_NEW_NAME + "/']"));
+        Assert.assertEquals(deletedProjPage.getTitle(), "Dashboard [Jenkins]");
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/MultiConfigurationTest.java:113]: As far as you remember, Sergey said that using the getTitle method in Assert is a bad practice.

**Review by evanmaiqa — COMMENTED** [review]
The name of the PR must match the name of the Trello card.

File: src/test/java/school/redrover/MultiConfigurationTest.java (hunk 1 above)
Comment (goluublue) [reply src/test/java/school/redrover/MultiConfigurationTest.java:113]: As far as I remember, Sergey's issue was with the getURL method, not getTitle.
Either way, I was just converting over an existing assert; I can delete it in the next refactor pass.

**Review by ArtyomDulya — APPROVED** [review]

**Pull request merged**